### PR TITLE
Fix the 1.54.0 release

### PR DESCRIPTION
**Pull request by nex3**

### CODE REVIEW

**Comment by ntkme** [comment]
How about a matrix build to make it a little bit easier to maintain? Something like this (untested):

``` yaml
  deploy_github_macos:
    name: "Deploy Github: Mac OS"
    runs-on: ${{ matrix.runner }}
    needs: [deploy_github_linux]
    strategy:
      matrix:
        include:
          - arch: x64
            runner: macos-latest
          - arch: arm64
            runner: self-hosted
    if: "startsWith(github.ref, 'refs/tags/') && github.repository == 'sass/dart-sass-embedded'"
    steps:
      - uses: actions/checkout@v2
      - uses: arduino/setup-protoc@v1
        with: { version: "${{ env.protoc_version }}", repo-token: "${{ github.token }}" }
      - uses: dart-lang/setup-dart@v1
      - run: dart pub get
      - run: dart run grinder protobuf
      - name: Deploy
        run: dart run grinder pkg-github-macos-${{ matrix.arch }}
        env: {GH_BEARER_TOKEN: "${{ github.token }}"}
```

**Comment by nex3** [comment]
Good idea, let's give that a try.

**Review by Goodwine — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -268,28 +268,20 @@ jobs:
...
-      - run: dart pub get
-      - run: dart run grinder protobuf
-      - name: Deploy
-        run: dart run grinder pkg-github-macos
-        env: {GH_BEARER_TOKEN: "${{ github.token }}"}
-
-  deploy_github_windows:
-    name: "Deploy Github: Windows"
-    runs-on: windows-latest
+  deploy_github:
```
Comment (Goodwine) [inline .github/workflows/ci.yml:271]: > Job 'release_embedded_host' depends on unknown job 'deploy_github_macos'

**Review by Goodwine — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -234,7 +234,7 @@ jobs:
         env: {GH_BEARER_TOKEN: "${{ github.token }}"}
 
   deploy_github_linux_qemu:
-    name: "Deploy Github: Linux"
+    name: "Deploy Github: linux-arm64"
```
Comment (ntkme) [inline .github/workflows/ci.yml]:
```
    name: "Deploy Github: linux-${{ matrix.arch }}"
```

**Review by ntkme — CHANGES_REQUESTED** [review]

**Review by Goodwine — APPROVED** [review]

**Comment by ntkme** [comment]
@nex3 Looks like we need to manually copy the changelog to here in order to pass the version number check.

**Pull request merged**